collections/guides/manage/manage-reactor-formulas-terraform.md

```diff
@@ -0,0 +1,174 @@
+
+## Set up Terraform configuration
+
+Let's set up the Terraform configuration you'll need for creating a Reactor Formula.
+
+### Set up the basistheory provider
+
+Firstly, you'll need to configure the `terraform` and `provider` configuration blocks in a `main.tf` file. Let's add the configuration for the provider source and provider configuration.
+You can find the latest versions available and other configuration options in the provider documentation [here](https://registry.terraform.io/providers/Basis-Theory/basistheory).
+
+```terraform
+terraform {
+  required_providers {
+    basistheory = {
+      source  = "basis-theory/basistheory"
+      version = ">= 0.1.3"
+    }
+  }
+}
+
+provider "basistheory" {
+  api_key = "<YOUR_API_KEY_HERE>"
+}
+```
+
+### Set up the basistheory_reactor_formula resource
+
+Typically, resources are set up in a different file outside of `main.tf`. Let's create a new `reactor_formula.tf`
+file with the following configuration for our Reactor Formula. Feel free to modify the following configuration for your use case:
+
+<span class="base-alert info">
+  <span>
+    For more information about writing your own code for a Reactor Formula, check out [our guide](/guides/run-your-own-code-in-a-reactor/).
+  </span>
+</span>
+
+```terraform
+resource "basistheory_reactor_formula" "my_awesome_formula" {
+  name        = "My Awesome Reactor Formula"
+  description = "Do something awesome!"
+  type        = "private"
+  icon        = "data:image/png;base64, iVBORw0KGgoAAAANSUhEUgAAAAUAAAAFCAYAAACNbyblAAAAHElEQVQI12P4//8/w38GIAXDIBKE0DHxgljNBAAO9TXL0Y4OHwAAAABJRU5ErkJggg=="
+  code        = <<-EOT
+  module.exports = async function (req) {
+    // Do something with `req.configuration.SERVICE_API_KEY`
+
+    // Do anything here!
+
+    return {
+      raw: {
+        foo: "bar"
+      },
+      tokenize: {}
+    };
+  };
+  EOT
+
+  configuration {
+    name        = "SERVICE_API_KEY"
+    description = "Configuration description"
+    type        = "string"
+  }
+
+  request_parameter {
+    name        = "request_parameter_1"
+    description = "Request parameter description"
+    type        = "string"
+  }
+
+  request_parameter {
+    name        = "request_parameter_2"
+    description = "Request parameter description"
+    type        = "boolean"
+    optional    = true
+  }
+}
+```
+
+### Set up a basistheory_reactor resource from the Reactor Formula
+
+Next, we can create and configure a Reactor referencing the Reactor Formula ID of `basistheory_reactor_formula.my_awesome_formula.id`.
+Let's create a new `reactor.tf` file with the following configuration for our Reactor Formula.
+Feel free to modify the following configuration for your use case:
+
+```terraform
+resource "basistheory_reactor" "my_awesome_reactor" {
+  name          = "My Awesome Reactor"
+  formula_id    = basistheory_reactor_formula.my_awesome_formula.id
+  configuration = {
+    SERVICE_API_KEY = "key_abcd1234"
```

Comment:
How would someone not embed this api key in plaintext? It feels wrong to encourage users to do this.